packages/tools/guiEditor/src/workbenchEditor.tsx

```diff
@@ -191,69 +195,134 @@ export class WorkbenchEditor extends React.Component<IGraphEditorProps, IGraphEd
             }
         }
     };
+    switchExpandedState(): void {
+        this.setState({ toolbarExpand: !this.state.toolbarExpand });
+    }
 
     render() {
-        return (
-            <Portal globalState={this.props.globalState}>
-                <div id="ge-header">
-                    <div className="command-bar">
-                        <CommandBarComponent globalState={this.props.globalState} />
+        if (this.state.toolbarExpand) {

```

Comment:
is there a way to reuse the JSX here? like in the example I provided under (in `createToolbar`) it feels like you can reuse the code quite a lot here as well.